frontend/pages/dashboard/new-post.tsx

```diff
@@ -56,8 +94,10 @@ const NewPostPage: NextPage = () => {
     .concat(languagesNative.map((x) => x.language))
 
   const createNewPost = (status: PostStatusType) => {
+    const images = []
+    images.push(image)
     createPost({
-      variables: { title, body, status, languageId: langId },
+      variables: { title, body, status, languageId: langId, images },
```

Comment:
Just to cut down on line noise you can do:

```suggestion
      variables: { title, body, status, languageId: langId, images: [image] },
```

and get rid of lines 97-98

Reply:
Wonderful wee suggestion! Done!